src/boost/richstring.nim

```diff
@@ -52,7 +52,7 @@ proc handleSFormat(exp: string, fmtp: string): NimNode {.compileTime.} =
   if maxLen == 0:
     result = parseExpr("$(" & exp & ")")
   else:
-    result = newCall(bindSym"alignStr", parseExpr("$(" & exp & ")"), newLit(maxLen), newLit(' '))
+    result = newCall(bindSym"alignStr", parseExpr("$(" & exp & ")"), newLit(maxLen), newLit(' '), newLit(true))
```

Comment:
I thought it was on purpose to disable truncation by default in `fmt`. At least it would be a bit surprising coming from Scala, where `f"${"asdf"}%2s"` does not truncate.